File: 백준/Bronze/10818. 최소， 최대/최소， 최대.cc
#include <iostream>
using namespace std;

int main()
{
    int N;
    int max, min;
    cin>>N;
    int arr[N];
    
    for(int i = 0; i<N; i++)
    {
        cin>>arr[i];   
    }
    max=arr[0];
    min=arr[0];
    
    for(int j = 1; j<N; j++)
    {
        if(max < arr[j])
        {
            max=arr[j];
        }
        if(min > arr[j])
        {
            min=arr[j];
        }
    }
    cout<<min<<" "<<max;
    return 0;
}

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int N = sc.nextInt();
		int Max = -1000000;
		int Min = 1000000;
		for(int i=0; i<N; i++) {
			int num = sc.nextInt();
			if(Max < num) {
				Max = num;
			}
			if(Min > num) {
				Min = num;
			}
		}
		System.out.print(Min + " ");
		System.out.print(Max);
	}
}